### PR TITLE
Cartopy v0.20 docs with updated version number

### PR DESCRIPTION
**Pull request by greglucas**

### CODE REVIEW

**Comment by greglucas** [comment]
ping @QuLogic. I grabbed your new branch build and it looks like that got the proper version numbers.

File: shared_assets/4669ebe50291ed20fcf5c790ee4f42cc3291db3cdc8f3d248b26d43a-version_switch.js [hunk 1]
```diff
@@ -5,8 +5,7 @@
   'use strict';
 
   var all_versions = {
-    'latest': '0.20',
-    'v0.19': '0.19',
+    'latest': '0.19',
```
Comment (QuLogic) [inline shared_assets/4669ebe50291ed20fcf5c790ee4f42cc3291db3cdc8f3d248b26d43a-version_switch.js:8]: Should probably revert this?

**Comment by QuLogic** [comment]
PS, no need to worry about rebasing/amending; this repo only allows squash merges apparently.

**Comment by greglucas** [comment]
:+1: We may move somewhere else for the docs in the future too.

**Pull request merged**